pages/linux/x11vnc.md

```diff
@@ -0,0 +1,23 @@
+# x11vnc
+
+> A VNC server that will enable VNC on an existing display.
+
+- Launch a VNC server that allows multiple clients to connect:
+
+`x11vnc -shared`
+
+- Launch a VNC server that will not exist after a client disconnects:
+
+`x11vnc -forever -viewonly`
```

Comment:
Ah, then your description is incorrect actually. You have said that the server will **not** exist adter a client disconnects. :wink:  

This is my version - 
"Launch the server where the user can only view the screen, and will continue to run even after the last client disconnects"